keras_cv/layers/preprocessing/jittered_resize.py

```diff
@@ -46,7 +50,26 @@ class JitteredResize(BaseImageAugmentationLayer):
     are translated and scaled according to the random scaling and random
     cropping.
 
+    Args:
+        target_size: A tuple representing the output size of images.
+        scale_factor: A tuple of two floats or a `keras_cv.FactorSampler`. For
+            each augmented image a value is sampled from the provided range.
+            This factor is used to scale the input image.
+            To replicate the results of the MaskRCNN paper pass `(0.8, 1.25)`.
+        crop_size: (Optional) the size of the image to crop from the scaled
+            image, defaults to `target_size` when not provided.
+        bounding_box_format: The format of bounding boxes of input boxes.
+            Refer to
+            https://github.com/keras-team/keras-cv/blob/master/keras_cv/bounding_box/converters.py
+            for more details on supported bounding box formats.
+        interpolation: String, the interpolation method, defaults to
+            `"bilinear"`. Supports `"bilinear"`, `"nearest"`, `"bicubic"`,
+            `"area"`, `"lanczos3"`, `"lanczos5"`, `"gaussian"`,
+            `"mitchellcubic"`.
+        seed: (Optional) integer to use as the random seed.
+
     Usage:
+
```

Comment:
its weird but remove newline here - the code block has some weird behavior on keras.io

Reply:
Done.